js/user.js

```diff
@@ -6,4 +6,8 @@ function User(vkUser) {
 	this.getFullName = function() {
 		return user.first_name + " " + user.last_name;
 	};
+	this.getFriends = function() {

```

Comment:
friends shouldn't be a part of User object. Create separate js file for "friends" functionality where you'll wrap call to VK API